assembly/fabric8-ide-gwt-app/src/main/resources/org/eclipse/che/public/keycloak/RhCheKeycloak.js

```diff
@@ -406,103 +456,134 @@ function initAnalytics(writeKey){
 
             setStatusMessage("");
 
-            var promise = originalInit(initOptions);
-            promise.success(function(arg) {
-                var keycloak = kc;
-                var lastProvisioningDate = sessionStorage.getItem('osio-provisioning');
-                sessionStorage.removeItem('osio-provisioning');
-                if (lastProvisioningDate) {
-                    var w = window.open('', 'osio_provisioning');
-                    w && w.close();
+            const segmentWriteKeyUrl = '/api/fabric8-che-analytics/segment-write-key';
+            get(segmentWriteKeyUrl)
+            .then(function (request) {
```

Comment:
arrow function might be used as well

function (request)

==>

(request) => {}

Reply:
fixed